lib/hawkbit/hawkbit_priv.h

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (c) 2020 Linumiz
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** @file
+ *
+ * @brief This file contains structures representing JSON messages
+ * exchanged with a hawkbit
+ */
+
+#ifndef __HAWKBIT_PRIV_H__
+#define __HAWKBIT_PRIV_H__
+
+#include <data/json.h>
+
+#define HAWKBIT_SLEEP_LENGTH 8
+
+enum hawkbit_http_request {
+	HAWKBIT_PROBE = 0,
+	HAWKBIT_CONFIG_DEVICE,
+	HAWKBIT_CLOSE,
+	HAWKBIT_PROBE_DEPLOYMENT_BASE,
+	HAWKBIT_REPORT,
+	HAWKBIT_DOWNLOAD,
+};
+
+enum hawkbit_status_fini {
+        HAWKBIT_STATUS_FINISHED_SUCCESS = 0,
```

Comment:
something weird with indentation here or above, they should be aligned same ways but do not atm